extensions/icons/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## Add Icons (2022-03-24)
+Initial version
```

Comment:
```suggestion
## [Add Icons] - 2022-03-24
- Initial version
```

https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history